src/core/util.cpp

```diff
@@ -754,31 +754,47 @@ namespace mamba
     LockFile::LockFile(const fs::path& path)
         : m_path(path)
     {
-        LOG_INFO << "Locking " << path;
+        if (!fs::exists(path))
```

Comment:
should we only check existence if the it's a directory, or check if the parent directory exists?

Reply:
I don't really see. Here I want to check existence for both directory or file